### DISCUSSION TITLE
現在のアドベントカレンダーの参加者数の確認方法

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#71] [ヘルプ] [answered]
以前までアドベントカレンダーのページが各年の参加者数とカレンダー数の一覧ページだったと思いますが、現在該当のアドベントカレンダーのURLが2021年のアドベントカレンダーのトップページになっていると思います。そのため現在のアドベントカレンダーの参加者数の確認方法が分からない・・・のですが何らか確認する方法はございますでしょうか・・・？

巨大Qiitanぬいぐるみの件が流れてきて興味程度に参加者数の推移を見てみたい・・・と思った次第です！

**Comment by kskwtnk** [comment, marked as answer]
@simon-ritchie 

ご意見ありがとうございます！
現在、サイト上で確認できるページはないのですが、来週月曜日から[Twitterの公式アカウント](https://twitter.com/Qiita)で「現在の人数」をアナウンスすることになりました！

残り何人！といったアナウンスになると思うので、よろしければチェックしてみてください 🙏

**Comment by simon-ritchie** [comment]
@xrxoxcxox さん

> 現在、サイト上で確認できるページはないのですが、来週月曜日からTwitterの公式アカウントで「現在の人数」をアナウンスすることになりました！

承知いたしました！ご共有ありがとうございます！mm

**Reply by kskwtnk** [reply]
@simon-ritchie 

遅ればせながらですが……

https://twitter.com/Qiita/status/1465259239920730120

こういった感じで、Advent Calendarについてのツイートの中で人数のアナウンスを実施してきます！というご報告です！

**Reply by simon-ritchie** [reply]
@xrxoxcxox さん

おお・・・記憶が確かならもう前年は超えて、前々年の一番参加者が多かった年に肉薄しているのですね・・・！
ご共有ありがとうございます！

**Reply by kskwtnk** [reply]
@simon-ritchie 

まさにその通りなな推移でして、記憶力にびっくりしています笑
こちらこそリクエストありがとうございました！
